easybuild/easyconfigs/g/GDAL/GDAL-2.2.3-foss-2018b-Python-3.7.0.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '2.2.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.gdal.org/'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://download.osgeo.org/gdal/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['a328d63d476b3653f5a25b5f7971e87a15cdf8860ab0729d4b1157ba988b8d0b']
+
+dependencies = [
+    ('Python', '3.7.0'),
+    ('netCDF', '4.6.1'),
+    ('expat', '2.2.5', '-GCCcore-7.3.0', True),
```

Comment:
@fizwit Why is this hardcoded to use `GCCcore`, there should be no need for this? Same below.